packages/components/src/Select/docs/error.tsx

```diff
@@ -0,0 +1,19 @@
+import { Select, Label, ErrorMessage } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Select 
+            fieldChildren={
```

Comment:
I'll say it once to avoid repeating, but if it's more than one line, I would probably extract it to a function or const for readability.